### DISCUSSION TITLE
Welcome to hacktoberfest-swag-list Discussions!

### DISCUSSION THREAD

**Discussion by monizb** [monizb/hacktoberfest-swag-list#1] [Announcements]
## 👋 Welcome to the Hacktoberfest 2021 Swag List!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Add some more awesome swags to the list
  * Share your repository links where you are a maintainer

**Comment by arzitmahajan** [comment]
How to resolve restyling issue? I have followed all steps but its still not resolving.
